CMakeLists.txt

```
@@ -94,3 +98,37 @@ if(QHOTKEY_INSTALL)
 
     export(TARGETS qhotkey FILE QHotkeyConfig.cmake)
 endif()
+
+if(UNIX)
+    # General settings
+    set(CPACK_GENERATOR "TBZ2")
+    set(CPACK_PACKAGE_NAME "libqhotkey")
```

Relevant only for Debian:
```suggestion
    set(CPACK_DEBIAN_PACKAGE_NAME "libqhotkey")
```